### ISSUE TITLE
Labware calibration: "Continue moving to tip rack" button sometimes does nothing

### ISSUE THREAD

**Issue by SyntaxColoring** [Opentrons/opentrons#4810]
# Current behavior

The "Continue moving to tip rack" button in labware calibration sometimes silently fails.

1. The user clicks the "Continue moving to tip rack" button during labware calibration.
2. The loading spinner briefly appears over the tip on the deck layout view, then disappears.
3. Nothing happens. The robot doesn't move. The user can't continue with labware calibration.

# Expected behavior

Either the robot should move to the tip rack, or we should see an error message explaining why the robot isn't moving to the tip rack.

# Possible causes

Opentrons employees, see this issue's Slack thread for case references.

## Deck calibration

Redoing deck calibration has resolved the problem for one customer.

## Internal `AttributeError`

This traceback comes from a customer's API log. This part seems to repeat every time the button is clicked.

```
[redacted] opentrons-api[339]: Moving p10_single_v1.3 to opentrons_96_tiprack_10ul in 6
[redacted] opentrons-api[339]: Exception during RPC call:
   Traceback (most recent call last):
    File "usr/lib/python3.7/site-packages/opentrons/server/rpc.py", line 270, in make_call
    File "usr/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    File "usr/lib/python3.7/site-packages/opentrons/server/rpc.py", line 260, in call_and_serialize
    File "usr/lib/python3.7/site-packages/opentrons/api/calibration.py", line 37, in decorated
    File "usr/lib/python3.7/site-packages/opentrons/api/calibration.py", line 53, in decorated
    File "usr/lib/python3.7/site-packages/opentrons/api/calibration.py", line 222, in move_to
    File "/buildroot/output/target/usr/lib/python3.7/contextlib.py", line 112, in __enter__
    File "usr/lib/python3.7/site-packages/opentrons/protocol_api/contexts.py", line 250, in temp_connect
  AttributeError: 'list' object has no attribute 'values'
```

# If you're an Opentrons customer who's seeing this

Please send us your **api.log** and **serial.log** files so we can investigate. (Go to **Advanced Settings > Download Logs** and click **Download.**)

The logs will show information about how your OT-2 is being used. If you don't want that publicized, you can email them to support@opentrons.com and mention this issue. Otherwise, you can upload them in this thread.

**Comment by mcous** [comment]
The app collects the error from the API and does not display it. We can display the error, but as far as the error message goes, while calibration processes are on the legacy RPC API they will be mostly, if not completely, non-user-actionable.

The most common error to cause this behavior will be fixed by #4802

**Comment by mcous** [comment]
Oh also, unrelated to this fix for this ticket, but in the logs referenced above, that specific bug was fixed in #4614 and released in 3.15.2

**Comment by SyntaxColoring** [comment]
Okay, so the issue here is that internal problems – whatever they might be – are silently swallowed instead of being displayed in an error dialog or whatever.

Rather than rewriting this ticket, I've created a new one: #5551. Closing this in favor of that.
